docs/core/upgrade.md

```diff
@@ -82,6 +82,32 @@ To learn more about configuring migration scripts for your modules, see the [Mig
 
 You can introduce entirely new modules to the application during an upgrade. New modules are recognized because they have not yet been registered in `x/upgrade`'s `VersionMap` store. In this case, `RunMigrations` calls the `InitGenesis` function from the corresponding module to set up its initial state.
 
+### Add StoreUpgrades for New Modules
+
+All chains preparing to run in-place store migrations will need to manually add store upgrades for new modules and then configure the store loader to apply those upgrades. This ensures that the new module's stores are added to the multistore before the migrations begin.
+
+```golang
+  upgradeInfo, err := app.UpgradeKeeper.ReadUpgradeInfoFromDisk()
+	if err != nil {
+		panic(err)
+	}
+
+	if upgradeInfo.Name == "my-plan" && !app.UpgradeKeeper.IsSkipHeight(upgradeInfo.Height) {
+		storeUpgrades := storetypes.StoreUpgrades{
+
+        // ...
+        // add store upgrades for new modules
+        // Example: 
+        //    Added: []string{"foo", "bar"},
+        // ...
+
```

Comment:
```suggestion
			// add store upgrades for new modules
			// Example: 
			//    Added: []string{"foo", "bar"},
			// ...
```

Reply:
I don't know why the comments are appearing this way here, but they are in good indentation if you checkout to this branch and see them